packages/core/src/colors/palette.ts

```diff
@@ -22,4 +26,10 @@ export interface Palette {
 
   accentDark: string;
   accentDarkForeground: string;
+
+  disabledBackground: string;
+  disabledForeground: string;
+
+  error: string;
+  onError: string;
```

Comment:
Does `onError` belong to palette?

Reply:
We currently only use `error`. It works for all our current components. However in the future we might add a `ux-notification` component for exemple (displaying notifications on top or bottom of screen). If I think about this, we might have success, warning and error notifications. For error notifications we might use the `error` color as a background and then need to know that color use `onError`. 

Btw the `on` prefix comes from material design guideline but is a bit confusing when we compare to aurelia color naming (with ...Background and ...Foreground).

Our options seems to be:
* remove `onError` for now and see later
* keep it as this
* find better naming to avoid confusion

Reply:
I see, thanks for this explanation. So let's ping @EisenbergEffect @serifine for this as well. It seems good just following the design guideline to me.

Reply:
The more I think about it, the more I think `onError` should be renamed `errorForeground` to follow the pattern of `primary` and `accent` naming.